csp-mixin/rows.libsonnet

```diff
@@ -171,96 +171,194 @@ local g = import './g.libsonnet';
         + g.panel.timeSeries.gridPos.withW(8)
         + g.panel.timeSeries.gridPos.withH(8),
       ],
 
       vn_packets: [
         g.panel.row.new('Packets'),
-        this.grafana.panels.vn_packets_by_action
+        this.grafana.panels.azurevirtualnetwork.vn_packets_by_action
         + g.panel.timeSeries.gridPos.withW(8)
         + g.panel.timeSeries.gridPos.withH(8),
 
-        this.grafana.panels.vn_packets_dropped_by_protocol
+        this.grafana.panels.azurevirtualnetwork.vn_packets_dropped_by_protocol
         + g.panel.timeSeries.gridPos.withW(8)
         + g.panel.timeSeries.gridPos.withH(8),
 
-        this.grafana.panels.vn_packets_forwarded_by_protocol
+        this.grafana.panels.azurevirtualnetwork.vn_packets_forwarded_by_protocol
         + g.panel.timeSeries.gridPos.withW(8)
         + g.panel.timeSeries.gridPos.withH(8),
       ],
+
+
+      // Azure Virtual Machines
+      avm_overview: [
+        g.panel.row.new('Overview'),
+
+        this.grafana.panels.azurevm.avm_instance_count
+        + g.panel.timeSeries.gridPos.withW(12)
+        + g.panel.timeSeries.gridPos.withH(5),
+
+        this.grafana.panels.azurevm.avm_availability
+        + g.panel.timeSeries.gridPos.withW(12)
+        + g.panel.timeSeries.gridPos.withH(5),
+
+        this.grafana.panels.azurevm.avm_top5_cpu_utilization
+        + g.panel.timeSeries.gridPos.withW(12)
+        + g.panel.timeSeries.gridPos.withH(8),
+
+        this.grafana.panels.azurevm.avm_bottom5_memory_available
+        + g.panel.timeSeries.gridPos.withW(12)
+        + g.panel.timeSeries.gridPos.withH(8),
+
+        this.grafana.panels.azurevm.avm_top5_disk_read
+        + g.panel.timeSeries.gridPos.withW(12)
+        + g.panel.timeSeries.gridPos.withH(8),
+
+        this.grafana.panels.azurevm.avm_top5_disk_write
+        + g.panel.timeSeries.gridPos.withW(12)
+        + g.panel.timeSeries.gridPos.withH(8),
+
+        this.grafana.panels.azurevm.avm_instances_table
+        + g.panel.timeSeries.gridPos.withW(24)
+        + g.panel.timeSeries.gridPos.withH(8),
+
+        this.grafana.panels.azurevm.avm_disk_total_bytes
+        + g.panel.timeSeries.gridPos.withW(12)
+        + g.panel.timeSeries.gridPos.withH(8),
+
+        this.grafana.panels.azurevm.avm_disk_operations
+        + g.panel.timeSeries.gridPos.withW(12)
+        + g.panel.timeSeries.gridPos.withH(8),
+
+        this.grafana.panels.azurevm.avm_network_total
+        + g.panel.timeSeries.gridPos.withW(12)
+        + g.panel.timeSeries.gridPos.withH(8),
+
+        this.grafana.panels.azurevm.avm_connections
+        + g.panel.timeSeries.gridPos.withW(12)
+        + g.panel.timeSeries.gridPos.withH(8),
+      ],
+
+      avm_instance: [
+        g.panel.row.new('Instance')
+        + g.panel.row.withCollapsed(true)
+        + g.panel.row.withPanels(
+          [
+            this.grafana.panels.azurevm.avm_cpu_utilization
+            + g.panel.timeSeries.gridPos.withW(12)
```

Comment:
What do you think of adding a text panel with some short guidance, similar to the GCP dashboard?
Commit from GCP https://github.com/grafana/jsonnet-libs/pull/1336/commits/f1aacad278d399a8c0f6ec8ecc0430b81421d63d

Reply:
sure